docs/tutorials/t-05.md

```diff
@@ -0,0 +1,116 @@
+## Objective
```

Comment:
Let move it to `docs/guide`. I remember we already have tutorial from 1 to 7... 
@robert-zaremba Could you confirm it?

Reply:
1. no, it should be in tutorials
2. yes, the tutorial part should be "8". so let's rename to `t-08.md"

Reply:
Both points have been addressed.